examples/poll/src/components/FormWrapper/FormWrapper.tsx

```diff
@@ -78,6 +74,7 @@ export default function FormWrapper({ setIsEditor }: FormWrapperProps) {
     () => isEditor && pollSelected !== null,
     [isEditor, pollSelected],
   );
+  const widgets = useMemo(() => getWidgets(intl), [locale, intl]);
```

Comment:
locale is not dependency require by getWidgets.

Reply:
No but i want to reload the memo if we change the language, otherwise it may keep the translations of the previous locale, i'm really doubting the intl dep is enougth..

Reply:
Ok it does works :+1:

Reply:
Yep you can retrieve the active locale through `intl.locale`